blob/azureblob/testdata/TestConformance/TestAs/verify_As_returns_false_when_passed_nil.yaml

```diff
@@ -8,7 +8,7 @@ interactions:
       Content-Length:
       - "11"
       User-Agent:
-      - go-cloud/blob/0.9.0 Azure-Storage/0.3 (go1.11.2; linux)
+      - go-cloud/blob/0.1.0 Azure-Storage/0.4 (go1.12; linux)
```

Comment:
Why did the version jump back `0.9.0`->`0.1.0`?

Reply:
See #1118.